src/snowcli/output/printing.py

```diff
@@ -155,40 +144,21 @@ def _get_format_type() -> OutputFormat:
     return OutputFormat.TABLE
 
 
-def _print_formatted(data: List[Dict]) -> None:
-    output_format = _get_format_type()
-    if output_format == OutputFormat.TABLE:
-        _print_table(data)
-    elif output_format == OutputFormat.JSON:
-        import json
-
-        print_json(json.dumps(data, cls=CustomJSONEncoder))
-    else:
-        raise Exception(f"Unknown {output_format} format option")
-
-
-def print_data(data: List[Dict], columns: Optional[List[str]] = None) -> None:
-    if columns is not None:
-        data = [{k: v for k, v in raw.items() if k in columns} for raw in data]
-    _print_formatted(data)
-
-
-def _print_table(data: Union[str, Dict, List[Dict]]) -> None:
-    from rich.table import Table
-
-    if not data:
-        print("No data")
-        return
-
-    if isinstance(data, str):
-        print(data)
-        return
+def _render_table_output(data: OutputData) -> None:
+    stream = data.get_data()
+    for item in stream:
+        if isinstance(item, OutputData):
+            _render_table_output(item)
+        else:
+            _print_table(item, stream)
 
-    columns = list(data[0].keys())
 
+def _print_table(item, stream):
     table = Table(show_header=True, box=box.ASCII)
-    for column in columns:
+    for column in item.keys():
         table.add_column(column)
-    for row in data:
-        table.add_row(*[str(i) for i in row.values()])
-    print(table)
+    with Live(table, refresh_per_second=4):
+        table.add_row(*[str(i) for i in item.values()])
+        for item in stream:
+            table.add_row(*[str(i) for i in item.values()])
+    print("")
```

Comment:
This print can be removed?

Reply:
Let me check, it had some purpose to be aligned with previous tests 😄